package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "internships",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "core-js": "^3.6.5",
+    "vue": "^3.0.0",
+    "vue-router": "^4.0.0-0",
+    "vuex": "^4.0.0-0"
+  },
+  "devDependencies": {
```

Comment:
Why some parts of dependencies are in `dependencies` and the second part in `devDependencies`?

Reply:
Some of dependencies are not used in production. They're marked as `devDependencies` to not be installed by npm while they're not neccessary.

Reply:
@gLenczuk, I know that, but I don't know if Bartek knows why. ;)